activestate.yaml

```diff
@@ -138,17 +138,17 @@ scripts:
     value: |
       set -e
       echo "Building State Tool"
-      $scripts.build.path()
+      $scripts.build.bash-path()
```

Comment:
Hmm what if we did `$scripts.build.path(posix)`, or `$scripts.build.path.posix()`.

I don't think we should use `bash`, that's too opinionated.

Reply:
This, in essence, is creating a portion of the API that we'll need to continually support in order to inject some context into the expander. Further, this is an easy-to-miss dependency. For example, if a bunch of Linux users don't use path.posix() and then a Windows user joins the team and unexpectedly gets the wrong value. It's not intuitive to need to repeat bash/posix while inside of a bash script.

I very much think that we, in the go code, should change the working directory to the one holding the script, then get the path to the script using the shell determined by the script language. This solution would require us to be able to send in contextual info (the shell type) to the expander logic.

Reply:
As I said yesterday this can be an iteration, I agree the ideal we're striving for is that `path()` just handles this intelligently, but there are always edge cases and having explicit behavior "as well" does not hurt.

Reply:
A "POSIX" version of a Windows path is not what is needed. The resolved path is environment-dependent. For example, `C:\Users\username` is `/mnt/c/Users/username` in WSL Bash on Windows. This would be `/c/Users/username` if "bash" is actually git-bash. The context that is needed in the expander is which shell is being used so that that shell's environment can be called to get the path. A simple "conversion to posix" is not sufficient.